src/Features/CSharp/Portable/CodeRefactorings/SyncNamespace/CSharpSyncNamespaceService.cs

```diff
@@ -0,0 +1,294 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings.SyncNamespace;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.LanguageServices;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeRefactorings.SyncNamespace
+{
+    [ExportLanguageService(typeof(ISyncNamespaceService), LanguageNames.CSharp), Shared]
+    internal sealed class CSharpSyncNamespaceService :
+        AbstractSyncNamespaceService<NamespaceDeclarationSyntax, CompilationUnitSyntax>
+    {
+        protected override IReadOnlyList<SyntaxNode> GetMemberDeclarationsInContainer(SyntaxNode compilationUnitOrNamespaceDecl)
```

Comment:
this should return `SyntaxList<TMemberDeclarationSyntax>`

Reply:
it feels more precise as to what you're actually intendin ot return.  When i see "IReadOnlyList" there's an open question of "is this returning some strange subset of member decls?  Whereas, SyntaxList helps really just convey you're passing along the true syntax-list member of the node passed in. 

It's not a big deal, but it's more idiomatic with how most of the LS works.